l1-contracts/contracts/state-transition/IStateTransitionManager.sol

```diff
@@ -131,4 +131,6 @@ interface IStateTransitionManager {
         uint256 _oldProtocolVersion,
         Diamond.DiamondCutData calldata _diamondCut
     ) external;
+
+    function getSemverProtocolVersion() external view returns (uint32, uint32, uint32);
```

Comment:
Shall we create human-readable semver getter function?

Reply:
This will eliminate error from the offchain viewers to understand what order they should expect, wdyt?

Reply:
Talked in private. Decided that while it offers UX benefits, it may be worthwhile to add it in some of the future upgrades, since it will involve including a relatively big external library and the STM's bytecode is already large. Also, to reduce the diff